src/core/database/storage/NetworkModelStorage.ts

```diff
@@ -24,6 +30,15 @@ export class NetworkModelStorage extends VersionedNetworkBasedObjectStorage<Netw
   public static INSTANCE = new NetworkModelStorage()
 
   private constructor() {
-    super('networkCache')
+    super('networkCache', [
+      {
+        description: 'Update networkCache to 0.9.5.1 network',
+        migrate: (from: any) => {
+          // remove all pre-0.9.5.1 networks
+          const networks = Object.keys(from)
+          networks.map((networkGenHash) => resetNetwork(networkGenHash))
```

Comment:
The migration is a standalone function that converts from the old value to a new value. It does not need services (storage should not depend on higher levels components like services)

In this case, because you just want to clear out the table the function is:

`migrate: (from: any) => return undefined`

undefined means the data is going to be removed

https://github.com/nemfoundation/symbol-desktop-wallet/blob/master/src/core/database/backends/VersionedObjectStorage.ts#L57

Reply:
yes i will apply 0.19.2 before release :+1:

Reply:
ok thanks for the notes on migration

Reply:
fixed with e12ff0fa